src/elements/fields/TextField.jsx

```diff
@@ -89,6 +90,11 @@ function TextField({
 }) {
   const servar = element.servar;
   const inputType = fieldProps.as === 'textarea' ? 'textarea' : 'input';
+
+  useHotkeys('enter', (e) =>
```

Comment:
when is this event triggered? before we would specify the element on which we're listening to keydown but now that it's in a use effect, what happens?

Reply:
useHotKeys hook will listen for all key interactions on the element. When the text field is in focus, the hook will capture the key events and call the callback function(2nd argument). This technically replaces the onKeyDown event handler.